example/deploy/operator.yaml

```diff
@@ -14,7 +14,7 @@ spec:
     spec:
```

Comment:
What's the difference between example/deploy/operator.yaml and deploy/operator.yaml?

Reply:
deploy/operator would only deploy the ansible-operator. example deploy/operator deploys the example operator.

Reply:
Interesting, the definitions look pretty much the same except for a port

Reply:
Ah never mind, I missed the image name change